src/main/io/displayport_srxl.c

```diff
@@ -60,6 +61,20 @@ static int srxlWriteString(displayPort_t *displayPort, uint8_t col, uint8_t row,
     return 0;
 }
 
+static int srxlWriteCharWithAttribute(displayPort_t *displayPort, uint8_t col, uint8_t row, uint8_t attr, uint8_t c)
```

Comment:
What is the reason we are introducing more indirection here - to me it would look cleaner if we modified the signature of `srxlWriteChar()` and `srxlWriteString()` instead and added `DISPLAYPORT_ATTR_NONE` when they are used locally.

Reply:
It was done to suppress visual noise. Is it better to modify the signature and modify every call to `srxWrite{Char|String}`?

Reply:
There are only half a dozen or so calls, so I think yes. The layer of indirection that is introduced now will be confusing to somebody looking at it some time from now.